plutus-playground-client/src/Action.purs

```diff
@@ -313,7 +313,7 @@ validationFeedback errors =
 addWaitActionPane :: forall p. Tuple String (HTML p Query)
 addWaitActionPane =
   Tuple "add-wait" $
-    col4_
+    div [ classes [ col12, colSm6, colMd4 ] ]
```

Comment:
is it perhaps worth putting this in a helper, since it is quite common as a general layout piece?

Reply:
Yup, good idea. Pushed.